PhotoService/ClientApp/src/services/profile.service.js

```diff
@@ -12,8 +12,8 @@ class ProfileService {
         .catch((err) => {throw err.response.data})
     }
 
-    getUserCollections(username) {
-        return axios.get(API_URL+'collections', {params: {username}})
+    getUserCollections(username,publicOnly) {
+        return axios.get(API_URL+`collections/?username=${username}&publicOnly=${publicOnly}`)
```

Comment:
better to use params
```suggestion
        return axios.get(API_URL+`collections/?username=${username}&publicOnly=${publicOnly}`)
```